image.go

```diff
@@ -362,6 +472,87 @@ func (img *imageHDU) Write(data interface{}) error {
 	return err
 }
 
+// ToImage returns a pure Go image.Image instance from the FITS image.
+func (img *imageHDU) ToImage() image.Image {
+
+	var returnedImage image.Image
+
+	// Getting the HDU bitpix and axes.
+	header := img.Header()
+	bitpix := header.Bitpix()
+	axes := header.Axes()
+	raw := img.Raw()
+
+	width := axes[0]
+	height := axes[1]
+
+	log.Println("bitpix:", bitpix)
+	log.Println("width:", width)
+	log.Println("height:", height)
+
+	switch bitpix {
+	case 8:
+		//Gray
+		min := image.Point{X: 0, Y: 0}
+		max := image.Point{X: width, Y: height}
+
+		imgGray := image.NewGray(image.Rectangle{Min: min, Max: max})
+		imgGray.Pix = raw
+
+		returnedImage = imgGray
+	case 16:
+		//Gray16
+		min := image.Point{X: 0, Y: 0}
+		max := image.Point{X: width, Y: height}
+
+		imgGray16 := image.NewGray16(image.Rectangle{Min: min, Max: max})
+		imgGray16.Pix = raw
+
+		returnedImage = imgGray16
+	case 32:
+		//RGBA
+		min := image.Point{X: 0, Y: 0}
+		max := image.Point{X: width, Y: height}
+
+		imgRGBA := image.NewRGBA(image.Rectangle{Min: min, Max: max})
+		imgRGBA.Pix = raw
+
+		returnedImage = imgRGBA
+	case 64:
+		//RGBA64
+		min := image.Point{X: 0, Y: 0}
+		max := image.Point{X: width, Y: height}
+
+		imgRGBA := image.NewRGBA64(image.Rectangle{Min: min, Max: max})
+		imgRGBA.Pix = raw
+
+		returnedImage = imgRGBA
+	case -32:
+		//RGBA
+		min := image.Point{X: 0, Y: 0}
+		max := image.Point{X: width, Y: height}
+
+		imgRGBA := newF32Image(image.Rectangle{Min: min, Max: max})
+		imgRGBA.Pix = raw
+
+		returnedImage = imgRGBA
+	case -64:
+		//RGBA64
+		min := image.Point{X: 0, Y: 0}
+		max := image.Point{X: width, Y: height}
+
+		imgRGBA := newF64Image(image.Rectangle{Min: min, Max: max})
+		imgRGBA.Pix = raw
+
+		returnedImage = imgRGBA
+
+	default:
+		log.Fatal("invalid image type [bitpix=%d]", bitpix)
```

Comment:
it is customary to prepend the name of the package in such error/panic messages.
something like that:

```go
panic(fmt.Errorf("fitsio: image with unknown BITPIX value of %d", bitpix))
```